components/integration/use-cases/UseCases.js

```diff
@@ -56,7 +56,7 @@ export default function UseCases(props) {
                                 <div className={styles.cardDescription}>
                                     <p>Automation is just code after all - write a main() and do whatever you want.</p>
 
-                                    <p>Use Ballerina&apos;s extended standard library to connect to any system, speak any protocol, process any data, and run it anywhere (on a VM, in Kubernetes, or just as a script).</p>
+                                    <p>Use Ballerina&apos;s extended Ballerina library to connect to any system, speak any protocol, process any data, and run it anywhere (on a VM, in Kubernetes, or just as a script).</p>
```

Comment:
incorrect to say `extended Ballerina library`

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/7834/.